app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,46 @@
+class AnswersController < ApplicationController
+
+  def new
+    @answer = Answer.new
+    @question = Question.find(params[:id])
+  end
+
+  def create
+    @question = Question.find(params[:question_id])
+    @id = session[:user_id]
+
+
+    @answer = Answer.create(user_id: session[:user_id] , description: params[:description], question_id: params[:question_id])
+
+    p session[:user_id]
+    p @answer
+    redirect_to question_path(@question)
+  end
+
+  def show
+  end
```

Comment:
If you have empty actions you can delete them - rails will just render the associated view when the routes hit them.